rbac/management/relation_replicator/logging_replicator.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""RelationReplicator which writes to the outbox table."""
+
+import logging
+
+from kessel.relations.v1beta1 import common_pb2
+from management.relation_replicator.relation_replicator import RelationReplicator, ReplicationEvent
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class LoggingReplicator(RelationReplicator):
```

Comment:
What if we want to log relations and but also replicated them as well at same time ?

Reply:
Yeah that might be something we could do in the future, or just add logging to the other replicators. This is mainly for compatibility with existing behavior because tests relied on this logging format.